tests/features/addCountry.feature

```diff
@@ -0,0 +1,14 @@
+Feature: add country
+    As an admin
+    I want to add a new country to the countries list
+    So that I can add a new country that is not listed
```

Comment:
```suggestion
    I want to add a new country list
    So that I can view and manage them
```

Reply:
But the feature is about adding a new country and not managing them.